microsoft/testsuites/network/networksettings.py

```diff
@@ -0,0 +1,141 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from assertpy import assert_that
+
+from lisa import Node, TestCaseMetadata, TestSuite, TestSuiteMetadata
+from lisa.tools import Ethtool
+from lisa.util import LisaException
+
+
+@TestSuiteMetadata(
+    area="network",
+    category="functional",
+    description="""
+    This test suite runs the ethtool related network test cases.
+    """,
+)
+class NetworkSettings(TestSuite):
+    @TestCaseMetadata(
+        description="""
+            This test case verifies if ring buffer settings can be changed with ethtool.
+
+            Steps:
+            1. Get the current ring buffer settings.
+            2. Change the rx and tx value to new_values using ethtool.
+            3. Get the settings again and validate the current rx and tx
+                values are equal to the new_values assigned.
+            4. Revert back the rx and tx value to their orginal values.
+
+        """,
+        priority=1,
+    )
+    def validate_ringbuffer_settings_change(self, node: Node) -> None:
+        ethtool = node.tools[Ethtool]
+        devices_settings = ethtool.get_all_device_ring_buffer_settings()
+
+        for interface_settings in devices_settings:
+            interface = interface_settings.device_name
+            original_rx = int(interface_settings.current_ring_buffer_settings["RX"])
+            original_tx = int(interface_settings.current_ring_buffer_settings["TX"])
+
+            modified_rx = 10011
```

Comment:
The variable names can be `expected_rx`. It's clearer in test case what's expected or actual.

Reply:
1. It's better to explain why those values are selected.
2. Is it possible the original value is the same as this one? If so, may be the test doesn't change values actually.